### PR TITLE
Feature/df 401 add qualification property

### PR DESCRIPTION
**Pull request by fabiangotzen**

### CODE REVIEW

File: domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java [hunk 1]
```diff
@@ -76,27 +80,32 @@ public WorkExperience getWorkExperience() {
         return workExperience;
     }
 
-    public String getEducationCode() {
-        return educationCode;
+    public Qualification getQualificationCode() {
```
Comment (cjpsycos) [inline domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java]: Where is the getter for educationCode

File: web/src/main/resources/liquibase/changelog/20181212080000_add_qualification_property.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+    <changeSet author="fago" id="20181214080000-1">
+        <addColumn tableName="JOB_ADVERTISEMENT_OCCUPATION">
+            <column name="QUALIFICATION_CODE"
+                    type="VARCHAR(20)">
```
Comment (cjpsycos) [inline web/src/main/resources/liquibase/changelog/20181212080000_add_qualification_property.xml]: VARCHAR(10)

File: domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java (hunk 1 above)
Comment (fabiangotzen) [reply domain/src/main/java/ch/admin/seco/jobs/services/jobadservice/domain/jobadvertisement/Occupation.java]: It was just below. I've changed the order of the getters now.

File: web/src/main/resources/liquibase/changelog/20181212080000_add_qualification_property.xml (hunk 2 above)
Comment (fabiangotzen) [reply web/src/main/resources/liquibase/changelog/20181212080000_add_qualification_property.xml]: changed.

**Pull request merged**